src/data/css-quiz.ts

```diff
@@ -1154,6 +1154,25 @@ const cssQuiz = [
       "The justify-content property is used to position the grid in the grid container along the x-axis.",
     Link: "https://www.freecodecamp.org/news/css-grid-tutorial-with-cheatsheet/#css-grid-architecture",
   },
+  {
+    Question: "In CSS, how to center a div",
+    Answer: "flexbox and margin:auto",
+    Distractor1: "flexbox and margin: 10px",
+    Distractor2: "flexbox and margin:50%",
+    Distractor3: "flexbox and margin:100%",
```

Comment:
I think we need to be more explicit here

```suggestion
    Answer: "use  display: flex; on the parent container and  margin: auto; for the child container",
    Distractor1: "use  display: flex; on the parent container and  margin: 20px; for the child container",
    Distractor2: "use  display: flex; on the parent container and  margin: 50px; for the child container",
    Distractor3: "use  display: flex; on the parent container and  margin: 45%; for the child container",
```